lib/haml_coffee_assets/action_view/resolver.rb

```diff
@@ -8,7 +8,7 @@ module ActionView
     # in it would normally be a fallback for all formats.
     #
     class Resolver < ::ActionView::FileSystemResolver
-      def find_templates(name, prefix, partial, details)
+      def find_templates(name, prefix, partial, details, outside_app_allowed = false)
```

Comment:
I think we will need to settle the question of the maintainer first, but I see your feedback is valid. I have written to Github to see what they do in these cases---  I believe what happens is they will promote one of the forks as the new root repo.